examlpes/offline_inference.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+import os
+import uuid
+import asyncio
+
+import ray
+from ray.util.queue import Queue as RayQueue
+from ray.util.scheduling_strategies import NodeAffinitySchedulingStrategy
+
+from llumnix import launch_ray_cluster, connect_to_ray_cluster, init_manager, init_llumlets
+from llumnix import (SamplingParams, ServerInfo, EngineManagerArgs, LLMEngineManager, Llumlet,
+                     EngineArgs, RequestOutput)
+
+# Sample prompts.
+prompts = [
+    "Hello, my name is",
+    "The president of the United States is",
+    "The capital of France is",
+    "The future of AI is",
+]
+
+# Create a sampling params object.
+sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
+
+# Launch ray cluster
+os.environ['HEAD_NODE'] = '1'
+os.environ['HEAD_NODE_IP'] = '127.0.0.1'
+ray_cluster_port=37000
+
+# Note: launch_ray_cluster will stop current ray cluster first, then init a new one.
+launch_ray_cluster(ray_cluster_port=ray_cluster_port)
+connect_to_ray_cluster(port=ray_cluster_port)
+
+# Set manager args and engine args
+manager_args = EngineManagerArgs()
+engine_args = EngineArgs(model="/mnt/cuikuilong.ckl/Qwen-7B", worker_use_ray=True,
```

Comment:
Do not specify model through path /mnt/...,  use facebook opt？

Reply:
done